shared/macros/10-warning.jinja

```diff
@@ -23,11 +23,17 @@
 :type rule: str
 :param release: Release number since the rule is officially deprecated.
 :type release: str
+:param is_rule: By default reason of deprecation is newer rule, but can be other circumstances
```

Comment:
What do you think about calling this new parameter by `reason_is_rule`? It will only be used when the macro is called with a reason which is not a rule. It feels more intuitive IMO. And the parameter description could be shortened in this case. e.g.: Set to False when `rule_or_reason` does not receive a rule as value.

Reply:
:+1: done in [c62844b](https://github.com/ComplianceAsCode/content/pull/10850/commits/c62844b1ba02ee621d08f8c34359f9ad2fbfd1c8)

Reply:
It seems you forgot to update the parameter in the description. : )

Reply:
:man_facepalming: